lingetic-nextjs-frontend/app/languages/page.tsx

```
@@ -25,20 +26,41 @@
+              name={language.name}
+              description={language.description}
+              image={language.image}
+            />
+          ))}
+        </div>
+
+        <div className="mt-16 text-center">
+          <p className="text-[#374151] mb-4">
+            Don't see the language you want to learn?
+          </p>
+          <button className="bg-white border-2 border-[#2563eb] text-[#2563eb] px-6 py-3 rounded-lg font-medium hover:bg-blue-50 transition-colors">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)